### PR TITLE
OCPBUGS-25647: runtimehandler: mixedcpus: fix quota calculation

### PR DESCRIPTION
**Pull request by Tal-or**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

Turns out the `podManager.GetCgroups()` doesn\'t return any information with regards to its existing resources.

We should put some extra effort in order to find and calculate the actual pod quota.
We need to sum the quota value from each container + the shared cpus and that should be our new pod quota.

#### Which issue(s) this PR fixes:
None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by Tal-or** [comment]
/hold 
More local testing are needed

File: internal/runtimehandlerhooks/high_performance_hooks_linux.go
Comment (haircommander) [inline internal/runtimehandlerhooks/high_performance_hooks_linux.go:1126]: this isn't present for v2

Comment (Tal-or) [reply internal/runtimehandlerhooks/high_performance_hooks_linux.go:1126]: you're right, I was focusing on the v1 fix.

Comment (Tal-or) [reply internal/runtimehandlerhooks/high_performance_hooks_linux.go]: Fixed for both

**Comment by Tal-or** [comment]
Tested on both v1 and v2

**Comment by haircommander** [comment]
/ok-to-test
/approve

thanks, @cri-o/cri-o-maintainers PTAL

**Comment by haircommander** [comment]
@cri-o/cri-o-maintainers PTAL

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**
